packages/oscal-react-library/src/components/oscal-utils/OSCALLinkUtils.ts

```diff
@@ -113,7 +113,7 @@ export function shiftFragmentSuffix(fragmentSuffix: string): string | undefined
  */
 export function conformLinkIdText(linkText: any): string {
```

Comment:
I would love if this didn't have to be `any`. The word `linkText` seems to imply this should be a `string`. Even the jsdoc says it is a string. But then we do `linkText.props.children`... 

Is it a small change to just make this a `string` and deal with those consequences?

Reply:
Only commenting on this since it was touched in this pr.

Reply:
Currently `linkText` can be anything to almost everything including button components and styling type things. I think if we restrict this to a string, some anchor links functionality will be slimmed down a bit. I see 2 ways of solving this problem:
1) Rename linkText to something like linkElement and fix the `@param` to `any`
2) Make this only a `string` type and remove unnecessary logic here and double check other places in the codebase that use this function. ` value` will need to be added to most AnchorLinks throughout the application.

Reply:
We can maybe do `string | ReactNode` for the type? Not 100% sure but let's try it?

Reply:
Oh I see why that would not work, but that seems very unsafe. I guess we can stick with `any` for now as I can't think of a cleaner solution at the moment.